tests/python_functional/tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py26,py27,py36,py37,py38
```

Comment:
On xenial we have py35. Would it make sense to add it too?

Reply:
+1 Sure